mne_bids_pipeline/_config.py

```diff
@@ -416,6 +416,14 @@
     ```
 """
 
+read_raw_bids_verbose: Optional[Literal["error"]] = None
```

Comment:
Added this one because I was getting a bunch of warnings when processing `ds000117`:
```
The search_str was "/mnt/bakraid/data/mne_data/ds000117-full/sub-01/**/meg/sub-01_ses-meg*channels.tsv"
  raw = read_raw_bids(
/home/larsoner/python/mne-bids-pipeline/mne_bids_pipeline/_import_data.py:226: RuntimeWarning: Did not find any meg.json associated with sub-01_ses-meg_task-facerecognition_run-03.

The search_str was "/mnt/bakraid/data/mne_data/ds000117-full/sub-01/**/meg/sub-01_ses-meg*meg.json"
```